Automattic/wp-calypso
# Social: Make the advanced product as the default one for social.

Related to p1676382513483749-slack-C02JJ910CNL. Merge this after the launch of the advanced plan. 

## Proposed Changes

* Show social advanced as the default product instead of the basic one. 

## Testing Instructions



* Boot jetpack cloud on local or use the live links. 
* Got http://jetpack.cloud.localhost:3000/pricing and click on Social and see that the default product selected is the advanced one. 

## Pre-merge Checklist
